chain/Cargo.toml

```diff
@@ -34,6 +33,7 @@ extindex = { version = "0.3", optional = true }
 
 # For tests
 tempfile = { version = "3.1", optional = true }
+tokio = { version = "0.2", features = ["test-util", "macros", "blocking", "rt-core", "rt-threaded"], default-features = false }
```

Comment:
Since it's only for tests, should make optional and add activate in `tests-utils` feature

Reply:
Do you duplicate the requirement in `dev-dependencies` then (`tempfile` is there twice following this scheme)?

Reply:
Can you test just putting it in `dev-dependencies` ? We may not need tokio in the `test-utils` dependencies if you switch to `exocore_core::futures::spawn_future`

Reply:
Indeed, I can build with `tokio` in `dev-dependencies` when using the wrapper